exercises/pangram/example.cpp

```diff
@@ -0,0 +1,30 @@
+#include "example.h"
+#include <cctype>
+namespace
+{
+    bool found_letter(char a, std::string s)
```

Comment:
Please add some extra vertical spacing (blank lines between the namespaces/functions/etc (see [word-count](https://github.com/exercism/cpp/blob/master/exercises/word-count/example.cpp))

And don't indent inside of the namespace block.